frontend/util/reports.js

```diff
@@ -91,3 +91,39 @@ export function plural(count, name) {
 export function relPath(url, baseurl) {
   return url.split(baseurl)[1];
 }
+
+export function getSuccessCriteria({ tags, id }) {
```

Comment:
set a default empty array `tags = []` incase the tags key is not an array passed to the function.